src/dbwrapper.cpp

```diff
@@ -159,12 +159,12 @@ bool CDBWrapper::WriteBatch(CDBBatch& batch, bool fSync)
     const bool log_memory = LogAcceptCategory(BCLog::LEVELDB);
     double mem_before = 0;
     if (log_memory) {
-        mem_before = DynamicMemoryUsage() / 1024 / 1024;
+        mem_before = (double)DynamicMemoryUsage() / 1024 / 1024;
```

Comment:
Wouldn't replacing 1024 with 1024.0 here (and in the other case) have the same effect, but avoid the cast?

Reply:
@laanwj Absolutely! Now updated. Please re-review :-)